src/test/java/core/basesyntax/service/impl/DataParserServiceImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParserService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class DataParserServiceImplTest {
+    private static final String HEADER = "type,fruit,quantity";
+    private static final String INVALID_OPERATION = "q";
+    private static final String INVALID_TRANSACTION_LINE = "q,apple,0";
+    private static final String VALID_TRANSACTION_LINE = "b,apple,80";
+    private static DataParserService dataParser;
+    private static List<String> dataToParse;
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        dataParser = new DataParserServiceImpl();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        dataToParse = new ArrayList<>();
+    }
+
+    @Test
+    public void parseDataToFruitTransaction_invalidOperationInput_notOk() {
```

Comment:
```suggestion
    public void parseData_invalidOperationInput_notOk() {
```
Because you renamed this method inside DataParser service